Purchases/Public/RCPurchases.m

```diff
@@ -925,8 +880,6 @@ - (void)applicationWillResignActive:(__unused NSNotification *)notif {
 }
 
 - (void)updateAllCachesIfNeeded {
-    // todo: move log to relevant class
-    [RCLog debug:[NSString stringWithFormat:@"%@", RCStrings.configure.application_active]];
```

Comment:
this one shouldn't have been here, but it was only called from `applicationDidBecomeActive` so it was fine.